src/main/java/com/ably/kafka/connect/DefaultChannelSinkMapping.java

```diff
@@ -0,0 +1,20 @@
+package com.ably.kafka.connect;
+
+import io.ably.lib.realtime.AblyRealtime;
+import io.ably.lib.realtime.Channel;
+import io.ably.lib.types.AblyException;
+import org.apache.kafka.connect.sink.SinkRecord;
+
+import javax.annotation.Nonnull;
+
+public class DefaultChannelSinkMapping implements ChannelSinkMapping {
+    private final ChannelSinkConnectorConfig sinkConnectorConfig;
+    public DefaultChannelSinkMapping(@Nonnull ChannelSinkConnectorConfig config) {
+        sinkConnectorConfig = config;
+    }
+
+    @Override
+    public Channel getChannel(@Nonnull SinkRecord sinkRecord, @Nonnull AblyRealtime ablyRealtime) throws AblyException {
+        return ablyRealtime.channels.get(sinkConnectorConfig.channelName, sinkConnectorConfig.channelOptions);
+    }
```

Comment:
Good question. This is because this is the first iteration of a bigger work. See https://github.com/ably/kafka-connect-ably/pull/38#issuecomment-1048706239
So basically this PR just creates the structure for new channel config and it's currently using the existing global config (static mapping). This will change when we add pattern based mapping